carbon-design-system/carbon
# fix(Modal): add proper right padding to header

Closes #13288 

Ensure Heading has proper padding alignment to match design specs shown here
![image](https://user-images.githubusercontent.com/9935383/228045457-276cf778-7092-455c-a97e-3402540231bf.png)

Currently, only the paragraph text has the 20% right padding layout

#### Changelog

**New**

- add 20% `padding-right` for the Header text to match the `p` text

**Changed**

- n/a

**Removed**

- n/a

#### Testing / Reviewing

1. Open Storybook > Modal > Default (or any version)
2. Inspect the `h3` Header and confirm there is now right padding
![image](https://user-images.githubusercontent.com/9935383/228045947-064387d9-eb0e-416a-8b09-46ebe3a1f559.png)
3. Also note that the padding values match the `p` tag below since they maintain an equal cutoff point (only IF there is a description area on that modal variation)
